framework/src/userobject/InterfaceUserObject.C

```diff
@@ -48,12 +48,58 @@ InterfaceUserObject::InterfaceUserObject(const InputParameters & parameters)
     _current_side_elem(_assembly.sideElem()),
     _current_side_volume(_assembly.sideElemVolume()),
     _neighbor_elem(_assembly.neighbor()),
-    _current_neighbor_volume(_assembly.neighborVolume())
+    _current_neighbor_volume(_assembly.neighborVolume()),
+    _fi(nullptr)
 {
   // Keep track of which variables are coupled so we know what we depend on
   const std::vector<MooseVariableFEBase *> & coupled_vars = getCoupledMooseVars();
   for (const auto & var : coupled_vars)
     addMooseVariableDependency(var);
+
+  // Check for finite volume variables
+  _fv = false;
+  for (const auto & var : coupled_vars)
+    if (var->isFV())
+      _fv = true;
```

Comment:
It seems like you are forcing a binary classification on something that might not be wholly finite volume or finite element. This seems somewhat trivial but maybe a rename from `_fv` to `_has_fv_vars` would help clarify this a bit?